src/card/info-status/info-status-card.component.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.card.component:pfInfoStatusCard
+ * @restrict E
+ *
+ * @param {object} status Status configuration information<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.title         - the main title of the info status card
+ * <li>.href          - the href to navigate to if one clicks on the title or count
+ * <li>.iconClass     - an icon to display to the left of the count
+ * <li>.iconImage     - an image to display to the left of Infrastructure
+ * <li>.info          - an array of strings to display, each element in the array is on a new line, accepts HTML content
+ * </ul>
+ * @param {boolean=} show-top-border Show/hide the top border, true shows top border, false (default) hides top border
+ * @param {boolean} htmlContent Flag to allow HTML content within the info options
+ *
+ * @description
+ * Component for easily displaying textual information
+ *
+ * @example
+ <example module="patternfly.card">
+
+ <file name="index.html">
+   <div ng-controller="CardDemoCtrl" style="display:inline-block;">
+     <div class="col-md-10">
+       <label>With Top Border, Icon Class, Href</label>
+       <pf-info-status-card status="infoStatus" show-top-border="true"></pf-info-status-card>
+       <br/>
+       <label>No Top Border, Icon Image, No Title</label>
+       <pf-info-status-card status="infoStatusTitless"></pf-info-status-card>
+       <br/>
+       <label>With HTML</label>
+       <pf-info-status-card status="infoStatusAlt" html-content="true"></pf-info-status-card>
+     </div>
+   </div>
+ </file>
+
+ <file name="script.js">
+   angular.module( 'patternfly.card' ).controller( 'CardDemoCtrl', function( $scope ) {
+    $scope.infoStatus = {
+      "title":"TinyCore-local",
+      "href":"#",
+      "iconClass": "fa fa-shield",
+      "info":[
+        "VM Name: aapdemo002",
+        "Host Name: localhost.localdomian",
+        "IP Address: 10.9.62.100",
+        "Power status: on"
+      ]
+    };
+
+    $scope.infoStatusTitless = {
+      "iconImage":"img/OpenShift-logo.svg",
+      "info":[
+        "Infastructure: VMware",
+        "Vmware: 1 CPU (1 socket x 1 core), 1024 MB",
+        "12 Snapshots",
+        "Drift History: 1"
+        ]
+    };
+
+    $scope.infoStatusAlt = {
+      "title":"Favorite Things",
+      "iconClass":"fa fa-heart",
+      "info":[
+        "<i class='fa fa-coffee'>",
+        "<i class='fa fa-motorcycle'>",
+        "<b>Tacos</b>"
+      ]
+    };
+   });
+ </file>
+
+ </example>
+ */
+
+angular.module( 'patternfly.card' ).component('pfInfoStatusCard', {
+  bindings: {
+    status: '=',
+    showTopBorder: '@?',
```

Comment:
Realize we use the ```@``` syntax related to strings in a few places related to Booleans. May have even used it on the referenced component. Not necessarily a fan of this syntax, especially the inevitable check that has to happen later in the controller ```someString === 'true'```. 

Would "one-way bind" syntax be better here? 

@jeff-phillips-18 @dtaylor113

Reply:
Oh snap, not a fan?  Well hot darn, I just did it cuz it was in place, less lines little simpler to just pass in `<?` gonna update pr to include that, 🙇  @cdcabrera

Reply:
We should be consistent with other cards. If we ever change it we should change it for all cards.

Reply:
That's an undertaking for sure, possibly breaking too for downstream if they're dumping in things like the curly brace syntax instead of... 

```html
<some-component a-boolean-attr="normalBoolean"
```
If this is a new element though implementation/consumption downstream wouldn't be as affected